modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -1,75 +1,81 @@
-|Yes 
+| Setting | Description | Required
+| Endpoint Name
+| The name used to identify the endpoint. The `Endpoint Name` is automatically generated, but can be modified manually.
+| Yes
 
-|Description
-|Describe any important details for this endpoint. 
+| Owner
+| The trading partner or host that owns the endpoint. 
+| Yes
+
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window. The value in this field cannot be modified.
+| Yes
+
+| Description
+| The user-defined field for describing the purpose and use of the endpoint.
 | No
 
 | AS2 URL
 a| The HTTP or HTTPS address where you receive AS2 messages.
 
-When deploying to CloudHub, the URL is automatically generated when you deploy a message flow that uses this endpoint.
-You can see the URL  by selecting the receiving endpoint from your deployed message flow.
-
-Select the receiving endpoint from your deployed message flow to view the URL.
+When deploying to CloudHub, the URL is automatically generated when you deploy a message flow that uses this endpoint. To view the URL, select the receiving endpoint from your deployed message flow.
 
-If you are deploying to on-premises Mule with a firewall and load balancer, work with your organization's hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule server is executing. You must replace the {runtime_host} value in the URL with the value that is specific to your organization. For example, if the DNS host name is `b2b.mycompany.com`, the AS2 receiver URL is: `+https://b2b.mycompany.com/receive-as2/+`
-|
+If you are deploying to on-premises Mule server with a firewall and load balancer, work with your organization's hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule server is executing. You must replace the {runtime_host} value in the URL with the value that is specific to your organization. For example, if the DNS host name is `b2b.mycompany.com`, the AS2 receiver URL is: `+https://b2b.mycompany.com/receive-as2/+`
+| No
 
 | <Host> public certificate
-a| This is your organization’s digital certificate, which is based on the RSA algorithm for the AS2 transport protocol.
-NOTE: If a single certificate is uploaded, Partner Manager prevents you from deleting your only certificate. 
-|Yes 
+| The digital certificate for your organization. This certificate is based on the RSA algorithm for the AS2 transport protocol.
+
+NOTE: You must have at least one certificate uploaded to the endpoint. If there is only one certificate uploaded to the endpoint, Partner Manager prevents you from deleting your only certificate.
+| Yes
 
 | <Host> private key file
-| This is your private key.
-|Yes 
+| The private key.
+| Yes
 
 | Key passphrase
-| This is the passphrase for your private key.
-| Yes 
+| The passphrase for your private key.
+| Yes
 |===
 
```

Comment:
BREAK and commit here to avoid GitHub freezing and to enable writer to get started with addressing comments.

Reply:
Got it!